lib/index.js

```diff
@@ -141,6 +141,21 @@
     return null;
   }
 
+  function configurePrivateNetworkAccess(options, req) {
+    if (!options.hasOwnProperty('allowPrivateNetworkAccess')) {
```

Comment:
The default value should just be added to https://github.com/expressjs/cors/blob/f038e7722838fd83935674aa8c5bf452766741fb/lib/index.js#L8 so no need to perform any type of default logic in this block.

Reply:
Solved.